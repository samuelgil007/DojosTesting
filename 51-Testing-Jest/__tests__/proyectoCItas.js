import Citas from "../js/classes/Citas";


describe("Pruebas para el proyecto de citas", () => {
    const citaClase = new Citas();
    const id =  909090;
    test("Agregar nueva cita", () => {

        
        const citaObj = {
            id: id,
            mascota: 'Rayo perro',
            propietario: 'Rayo dueño',
            telefono: '123',
            fecha: '10-10-2021',
            hora:'20:30',
            sintomas: 'prueba de sintomas'
        }
        citaClase.agregarCita(citaObj);

        expect(citaClase).toMatchSnapshot();
    });

    test("Actualizar cita", () => {

        
        const citaObj = {
            id: id,
            mascota: 'Rayo perro nuevo',
            propietario: 'Rayo dueño',
            telefono: '123',
            fecha: '10-10-2021',
            hora:'20:30',
            sintomas: 'prueba de sintomas'
        }
        citaClase.editarCita(citaObj);

        expect(citaClase).toMatchSnapshot();
    });

    test("Eliminar cita", () => {

        citaClase.eliminarCita(id);

        expect(citaClase).toMatchSnapshot();
    });
})