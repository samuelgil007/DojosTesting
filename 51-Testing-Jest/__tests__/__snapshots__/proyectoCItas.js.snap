// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas para el proyecto de citas Actualizar cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "10-10-2021",
      "hora": "20:30",
      "id": 909090,
      "mascota": "Rayo perro nuevo",
      "propietario": "Rayo dueño",
      "sintomas": "prueba de sintomas",
      "telefono": "123",
    },
  ],
}
`;

exports[`Pruebas para el proyecto de citas Agregar nueva cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "10-10-2021",
      "hora": "20:30",
      "id": 909090,
      "mascota": "Rayo perro",
      "propietario": "Rayo dueño",
      "sintomas": "prueba de sintomas",
      "telefono": "123",
    },
  ],
}
`;

exports[`Pruebas para el proyecto de citas Eliminar cita 1`] = `
Citas {
  "citas": Array [],
}
`;
